hugo_site/content/copenhagen.md

```diff
@@ -12,19 +12,19 @@ menu:
 
 ## København
 
-Copenhagen is a fantastic and beautiful city. Its well-designed urban infrastructure fascinates [milions of tourists](https://www.visitdenmark.dk/da/analyse/fakta-og-tal-om-turismen-i-danmark) every year. Since you are going to be one of them this time, here we lay out some of the basic things you need to get the most out of your stay in our city.
+Copenhagen is a fantastic and beautiful city. Its well-designed urban infrastructure and its [atractions](https://www.visitdenmark.com) fascinate [milions of tourists](https://www.visitdenmark.dk/da/analyse/fakta-og-tal-om-turismen-i-danmark) every year. Since you are going to be one of them this time, here we lay out some of the basic things you will need to get the most out of your stay in our city.
 
 To move around the capital - a quite small one -, we highly recommend you to rent a [bike](https://2019.djangocon.eu/bikes/). It is the most efficient and enviroment-friendly way to get to know things around here. Should you decide to use the public transport, that is not a problem! Below we have compiled the most important things you need to know about the public transport infrastructure.
 
 ## Transportation
 
-Denmark has a very functional and well-connected public transport. As many other cities in Europe, the Copenhagen is divided into [zones](https://dinoffentligetransport.dk/trafikinformation/trafikkort/zonekort/) and you pay as you move along these. Trains, buses and the metro all use the same ticket, so if you buy a ticket from Point A to Point B, you may mix differnt kinds of transportation veichles.
+Denmark has a very functional and well-connected public transport. As many other cities in Europe, Copenhagen is divided into [zones](https://dinoffentligetransport.dk/trafikinformation/trafikkort/zonekort/) and you pay as you move along these. Trains, buses and the metro all use the same ticket, so if you buy a ticket from Point A to Point B, you may mix differnt kinds of transportation veichles.
 
-Taxis (aka Taxa) in Denmark are quite expensive. A trip from the Kastrup / Copenhagen Airport to the city center will likely cost you around 400 Danish crowns (~60 USD). More information can be found [here](https://www.visitdenmark.com/denmark/transportation/taxis).
+Taxis (aka Taxa) in Denmark are quite expensive. A trip from the Kastrup / Copenhagen Airport to the city center will likely cost you around 400 Danish crowns (~60 USD) for a 10 KM trip. More information can be found [here](https://www.visitdenmark.com/denmark/transportation/taxis).
```

Comment:
Great to mention here that the metro is connected to the airport and to suggest that people use this instead?

Reply:
Maybe we should mention it in the Venue page. I didn't want to put this information, but IF someone would like to use Taxis, then there is something to read about here.

Reply:
Sounds good, we can probably add a separate "Getting here" section about ride-sharing/buses/trains/plains/airport.